<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="up_layout">
     <item>
      <layout class="QVBoxLayout" name="up_layout1">
       <item>
        <widget class="QLabel" name="iters_label">
         <property name="text">
          <string>Iterations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="iters_le"/>
       </item>
       <item>
        <widget class="QLabel" name="wheel_label">
         <property name="text">
          <string>Wheel sensitivity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="wheel_le"/>
       </item>
       <item>
        <widget class="QLabel" name="sizex_label">
         <property name="text">
          <string>Size x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sizex_le"/>
       </item>
       <item>
        <widget class="QLabel" name="sizey_label">
         <property name="text">
          <string>Size y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sizey_le"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="up_layout2">
       <item>
        <widget class="QLabel" name="init_hue_label">
         <property name="text">
          <string>Initial hue</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="init_hue_le"/>
       </item>
       <item>
        <widget class="QLabel" name="hue_cycle_label">
         <property name="text">
          <string>Hue cycle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hue_cycle_le"/>
       </item>
       <item>
        <widget class="QLabel" name="hue_dir_label">
         <property name="text">
          <string>Hue direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hue_dir_le"/>
       </item>
       <item>
        <widget class="QLabel" name="s_label">
         <property name="text">
          <string>Saturation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="s_le"/>
       </item>
       <item>
        <widget class="QLabel" name="v_label">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="v_le"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottom_layout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin" stdset="0">
      <number>0</number>
     </property>
     <item>
      <spacer name="bottom_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okBtn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
